### ISSUE TITLE
gem fails to load in windows with 1.9.2

### ISSUE THREAD

**Issue by bil-bas** [chipmunk-rb/chipmunk-ffi#15]
Using:
win7x64
Ruby 1.9.2 (from installer, but managed by pik)
ffi-1.0.9-x86-mingw32
chipmunk-ffi-1.1.0

When requiring 'chipmunk-ffi':

<pre>c:/Ruby192/lib/ruby/gems/1.9.1/gems/nice-ffi-0.4/lib/nice-ffi/library.rb:98:in `load_library':LoadError: Could not load chipmunk.</pre>

**Comment by shawn42** [comment]
Thanks for the issue:
where did you install the chipmunk.dll?

**Comment by bil-bas** [comment]
It loaded and seemed to work, as far as being able to create a new CP::Space, at least, when I just renamed libchipmunk.dll as chipmunk.dll. Duh!

However, I am positive that at some point in the past I had a much more horrid ffi error going on that I couldn't fix so easily. It also very occassionally worked but mostly didn't, and I had similar problems with other ffi things I tried. Can't remember what the issue was exactly. 

Since the chipmunk (non-ffi) gem comes as a binary or with auto-compiled source, it is actually easier to distribute, so I'll stick with it, now that someone is maintaining it again. Sorry if I mislead any potential users of your version about it _still_ not working on Windows!
